### DISCUSSION TITLE
sunbird-mobile-sdk integration with Diksha( Manage learn) mobile app

### DISCUSSION THREAD

**Discussion by vishwanath1004** [Sunbird-Ed/Community#605] [Installation]
I am trying to integrate sunbird-mobile-sdk version 6.0.1 with Sunbird-Ed-mobile-app. But am facing some issue so here i will explain the steps i followed:
Using MacBook.

In sunbird-mobile-sdk terminal
1. Cloned the sunbird-mobile-sdk with version release-6.0.1
2. npm i
3. npm run build:dev

In sunbird-client-service terminal
1. Cloned the sunbird-client-service with version release-6.0.0
2. npm i
3. npm run build:dev

In Mobile app terminal
1. Cloned App version 6.0
2. npm link  /Users/vishwanath/Documents/sunbird-mobile-sdk/tmp/
3. npm link /Users/vishwanath/Documents/sunbird-client-services/tmp
5. App build
6. Install the app in device
Issue is i am getting white screen  after splash screen  and no errors in console.
@swayangjit   Please help me  in this.

**Comment by vpPavithra** [comment]
@vishwanath1004  @swayangjit 
Before we link the sdk to mobile app, please follow the below steps
1. npm i crypto-browserify stream-browserify util
2. In pollyfills.ts
	import { Buffer } from 'buffer';
	global.Buffer = Buffer;	
	global.process = require('process');

3. npm run debug-sdk
then link sdk path. - npm link <sdk-path>

**Reply by vishwanath1004** [reply]
Hi @vpPavithra 
After that also its not working for me.

**Reply by chitranshu-keshav** [reply]
@vpPavithra Can you please get back on this
cc @ashokreddy1208
